aws/sdk/examples/ssm-put-parameter/src/main.rs

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use std::process;
+
+use ssm::model::ParameterType;
+use ssm::{Client, Config, Region};
+
+use aws_types::region::{EnvironmentProvider, ProvideRegion};
+
+#[tokio::main]
+async fn main() {
+    // Determine the region from environment variables or default to us-east-1
+    let region = EnvironmentProvider::new()
+        .region()
+        .unwrap_or_else(|| Region::new("us-east-1"));
+
+    // Construct a client
+    let config = Config::builder().region(region).build();
+    let client = Client::from_conf(config);
+
+    // Put a SSM application parameter named `test_parameter_name`
+    match client
```

Comment:
can you call get_parameter after this and retrieve it?

Reply:
(partially just to make sure that retrieval works)

Reply:
Added the following immediately after the put to test get:
```
    match client
        .get_parameter()
        .name("test_parameter_name")
        .send()
        .await
    {
        Ok(response) => {
            println!("Success! Response: {:?}", response)
        }
        Err(error) => {
            println!("Got an error getting the parameter: {}", error);
            process::exit(1);
        }
    }
```

And it works:
```
Success! Response: GetParameterOutput { parameter: Some(Parameter { name: Some("test_parameter_name"), r#type: Some(String), value: Some("some_value"), version: 4, selector: None, source_result: None, last_modified_date: Some(Instant { seconds: 1621441997, subsecond_nanos: 703999996 }), arn: Some("<omitted>/test_parameter_name"), data_type: Some("text") }) }
```